test/dune

```diff
@@ -0,0 +1,9 @@
+(executable
+  (name runner)
+  (flags :standard -w -3)
+  (modules :standard \ test_Unzip test_UTF8 test_Base64)
```

Comment:
They aren't tested now ?

Reply:
They haven’t been included in the standard distributions for quite a few versions already:
* `Base64` since 1.7.8 (released last year)
* `Unzip`, `UTF8` and `UChar` since 1.6.0 (released in 2013)

Reply:
Given they have not been part of the distribution (since everyone installs whatever OPAM ships with which is the "minimal distribution" which does not include these modules) I would suggest just deleting these modules and their tests. This makes the distribution simpler, the tests simpler and makes no difference for people using 1.7.8+.